full-testable-package.Dockerfile

```diff
@@ -76,6 +76,7 @@ COPY packages/data-broker/. ./packages/data-broker
 COPY packages/database/. ./packages/database
 COPY packages/devops/. ./packages/devops
 COPY packages/dhis-api/. ./packages/dhis-api
+COPY packages/entity-server/. ./packages/entity-server
```

Comment:
Required as currently `server-boilerplate` depends on `entity-server`, so now `entity-server` is an `internalDependency`